### PR TITLE
chore(test): add component test

### PR DESCRIPTION
**Pull request by doktordirk**
wip

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ module.exports = function (config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome', 'Firefox', 'ChromeCanary'],
+    browsers: ['Chrome'], //, 'Firefox', 'ChromeCanary'],
```
Comment (RWOverdijk) [inline karma.conf.js]: Is this PR still WIP, or am I allowed to mention this?

Comment (doktordirk) [inline karma.conf.js]: still WIP. i upped it in case you're interested in the howto

**Comment by doktordirk** [comment]
PS: if you meant like here, then it fine. I.ll Force push here till finished

**Comment by doktordirk** [comment]
have a basic version for milestone

**Comment by doktordirk** [comment]
still not much of use yet

**Pull request closed without merging**